source/databricks/calculation-engine/package/calculation/wholesale_calculation.py

```diff
@@ -0,0 +1,70 @@
+# Copyright 2020 Energinet DataHub A/S
+#
+# Licensed under the Apache License, Version 2.0 (the "License2");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from pyspark.sql import DataFrame
+from pyspark.sql.functions import col, when
+import package.calculation.wholesale.wholesale_initializer as init
+from package.calculation.wholesale.tariff_calculators import calculate_tariff_price_per_ga_co_es
+from package.codelists import ChargeResolution, MeteringPointType
+from package.constants import Colname
+from package.calculation_input import CalculationInputReader
+
+
+def execute(
+    calculation_input_reader: CalculationInputReader,
+    metering_points_periods_df: DataFrame,  # TODO: use enriched_time_series
+    time_series_point_df: DataFrame,  # TODO: use enriched_time_series
```

Comment:
Forgotten or planned for another task?
```suggestion
    metering_points_periods_df: DataFrame,  # TODO JMG: use enriched_time_series
    time_series_point_df: DataFrame,  # TODO JMG: use enriched_time_series
```

Reply:
planned